src/components/Navbar.jsx

```diff
@@ -37,15 +37,15 @@ const Navbar = () => {
     <>
       <nav className="navbar navbar-expand-lg navbar-light sticky-top navbar_main ">
         <div className="container">
-          <Link to={"/"}>
+          <Link className="Link" to={"/"}>
```

Comment:
Instead of the class name being `Link` can we make it something relevant? 
How about `nav_brand_parent`